app/forms/hyrax/forms/admin_set_form.rb

```diff
@@ -18,11 +17,8 @@ def [](key)
       end
 
       def permission_template
-        PermissionTemplateForm.new(@permission_template)
-      end
-
-      def workflow_name
-        @permission_template.workflow_name
+        template_model = PermissionTemplate.find_by!(admin_set_id: model.id)
+        PermissionTemplateForm.new(template_model)
```

Comment:
Is this something you want to memoize? And I noticed the `#workflow_name` method is being removed. Is that untested code?